content/en/docs/languages/go/_index.md

```diff
@@ -19,6 +18,8 @@ content:
 spelling: cSpell:ignore Isberner Klerk Malte youtube
 ---
 
+{{% docs/prog-lang-home-content %}}
+
 #### Developer stories and talks {#dev-stories}
```

Comment:
- [ ] Promote heading to level 3 (to match the other headings in the rendered page like "Learn more").
- [ ] Please also make this change in all other language-landing pages that have a similar heading.

Reply:
Updated!